### ISSUE TITLE
新手一枚，想请教一下作者最后怎么调用

### ISSUE THREAD

**Issue by Go24955** [plasavall/kdbscan_vdbscan#2]
from kdbscan import KDBSCAN
from vdbscan import VDBSCAN
from sklearn.datasets import load_iris
from scipy.spatial.distance import cosine

X = load_iris().data
alg1 = KDBSCAN(h=0.35,t=0.4,metric=cosine)
alg2 = VDBSCAN(kappa=0.005,metric=cosine)
kde = alg1.fit(X[:,1:3],return_kde = True)
alg2.fit(X,eta=0.5)
alg1_labels = alg1.labels_
alg2_labels = alg2.labels_
kdbscan.plot_kdbscan_results(kde)
在这个调用代码中会报错：最后一行的kdbscan未定义，不知要怎么解决，希望能得到回复，谢谢！
